src/app/pages/products/products-list/product-card/product-card.component.ts

```diff
@@ -9,27 +10,31 @@ import { WishlistService } from '@shared/services/wishlist.service';
 })
 export class ProductCardComponent implements OnInit {
     public stars: number[] = [1, 2, 3, 4, 5];
+    public existInWishlist: boolean;
 
     @Input() product: ProductInterface;
 
-    constructor(private wishlistService: WishlistService) {
+    constructor(private wishlistService: WishlistService, private notificationService: NotificationService) {
     }
 
     ngOnInit(): void {
+        this.isExistInWishlist();
     }
 
-    get existInWishlist(): ProductInterface {
-        return this.wishlistService.isExistInWishlist(this.product);
+    private isExistInWishlist(): void {
+        this.existInWishlist = this.wishlistService.isExistInWishlist(this.product);
```

Comment:
For the future...
If you see, that besides your `ProductInterface`, that you receive from BE or API, you will need in your project extra field, like `isAddedToWishList: boolean`, you can extand your interface with this field and than you can rid of this `wishlist array` in all your components, but your product will be included information if it was added to whishlist or no...